content/docs/introduction/technical-preview-free-tier.md

```diff
@@ -9,16 +9,18 @@ The Neon cloud service is available for free during the Technical Preview. For i
 
 The Technical Preview Free Tier has the following limits:
 
-- 1 Neon project
-- 10 branches including the [primary branch](/docs/reference/glossary#primary-branch), with a limit of 3GB of data per branch
-- A [compute endpoint](/docs/reference/glossary#compute-endpoint) for each branch, each with up to 1 vCPU and 4GB of RAM (with 512MB of RAM allocated to PostgreSQL shared buffers)
-- 100 Free Tier compute hours per month (total). Neon scales Free Tier computes to zero after 5 minutes of inactivity. Only active compute time is counted toward the Free Tier compute hour limit. When you reach the limit, Free Tier computes assigned to [non-primary branches](/docs/reference/glossary#non-primary-branch) are unavailable until the limit is reset at the beginning of the next month. You are always able to connect to the compute assigned to the primary branch, regardless of the limit. This ensures that access to data on the primary branch is never interrupted.
-- A point-in-time restore window of 7 days of _reasonable usage_
+- 1 Neon project.
+- 10 branches including the [primary branch](/docs/reference/glossary#primary-branch), with a limit of 3GB of data per branch.
+- A [compute endpoint](/docs/reference/glossary#compute-endpoint) for each branch, each with up to 1 vCPU and 4GB of RAM (with 512MB of RAM allocated to PostgreSQL shared buffers).
```

Comment:
If free tier compute resources are different than paid tier compute resources, we should be explicit about that.